src/customHooks/usePrice.js

```diff
@@ -0,0 +1,16 @@
+import { useCurrencyConverter } from "../customHooks";
```

Comment:
It is really cool to see you have used custom hooks for specific purposes. Learnt that from your code.Thanks